ArduSub/ArduSub.cpp

```diff
@@ -344,4 +346,14 @@ void Sub::update_altitude()
     }
 }
 
+void Sub::update_strobe_light()
+{
+    if(RC_Channels::get_radio_in(11) < 1500) {
+        return;
+    }
+
+    hal.rcout->write(12, strobe_state ? 1100 : g.strobe_max_bright);
```

Comment:
Maybe this is standard in Sub, but ideally this should go via SRV_Channels, adding a new Strobe function.

Reply:
Yes, we should think about it in sub to update this kind of call.